### PR TITLE
crud type

### PR DESCRIPTION
**Pull request by minhquang31011998**
## Related Tickets:
- #29472 https://edu-redmine.sun-asterisk.vn/issues/29472

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by minhquang31011998** [comment]
ready

**Comment by minhquang31011998** [comment]
Screenshot from 2020-10-13 21-50-26
Screenshot from 2020-10-13 21-50-29

File: app/Http/Controllers/Backend/TypeController.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        ->editColumn('id', function ($type) {
+            return '<div class="main__table-text">' . $type['id'] . '</div>';
+        })
+        ->editColumn('title', function ($type) {
+            return '<div class="main__table-text">' . $type['title'] . '</div>';
+        })
+        ->addColumn('action', function ($type) {
+            return
+                '<div class="main__table-btns">
+                    <a href="' . route('backend.type.edit', $type['id']) . '" class="main__table-btn main__table-btn--edit open-modal" data-toggle="tooltip" title="Edit">
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Backend/TypeController.php]: indent 4 spaces

**Comment by minhquang31011998** [comment]
ready

**Pull request merged**